wiki/Disambiguation/Ouendan/de.md

```diff
@@ -0,0 +1,13 @@
+---
+tags:
+  - Osu! Tatake! Ouendan
+---
+
+# Ouendan (Begriffserklärung)
+
+**Ouendan** kann Folgendes bedeuten:
+
+- Ein Cheerleader-Team (ähnlich zu Cheerleadern in der westlichen Kultur), welches ein Sportteam mit Trommeln, Fahnen und Banner anspornt und in japanischer Kultur zu finden ist.
```

Comment:
```suggestion
- Ein Cheerleader-Team (ähnlich zu Cheerleadern in der westlichen Kultur), welches ein Sportteam mit Trommeln, Fahnen und Bannern anspornt und in japanischer Kultur zu finden ist.
```

Reply:
The plural of `Banner` is `Banner`. There is [one case](https://de.wiktionary.org/wiki/Banner) in the `Dativ`, in which you write `den Bannern` ("Wem"). As far as I'm concerned, this is not applicable here. What do you think about it @Phippe?

Reply:
I think it's the same as saying `...welches ein Sportteam mit den Trommeln, den Fahnen und den Bannern anspornt...` (oder the classic `...welches ein Sportteam mit dem Mann anspornt...`), so it would be `Dativ` indeed.

[Here](https://www.deepl.com/translator#en/de/In%20Japanese%20culture%2C%20a%20cheer%20squad%20which%2C%20similar%20to%20western%20cheerleaders%2C%20rallies%20a%20sports%20team%20using%20drums%2C%20horns%2C%20banners%20and%20cheers%20of%20encouragement.) is the translation by DeepL, which also translates it to `Bannern`.